src/scripts/routes.js

```diff
@@ -544,21 +545,21 @@ import 'detailtablecss';
     });
 
     defineRoute({
-        path: '/configurationpage',
+        path: 'configurationpage',
```

Comment:
Currently, plugin configuration page opens directly - broken.
`page` has `/^configurationpage(?:\/(?=$))?$/i` regexp but requested url is `/configurationpage` (has leading `/`).
Route becomes unhandled and `page` just changes location.

Reply:
this relies on remote files to be loaded as modules. 

We could transform into json files and prep a base form builder in jf web so it could consume the serve json config files.

Reply:
forget what i said, you suggestion is enough.

Reply:
ahah sorry again, i was testing that part with the wrong link. It's still broken.

For those cases that remote files from the server are needed, i added a param to the route: serverRequest.

If this is present, i will fetch the full url from apiClient, make a get request, obtain the file, and proceed calling loadContent.

I already did and tested successfully.

Please check my changes and give your opinion.